README.md

```diff
@@ -218,6 +218,147 @@ You should create and keep your models under this folder. Note that we have appl
+  -h, --help            show this help message and exit
+  -b BLOCK_NUMBER, --block_number BLOCK_NUMBER
+                        Block number used for the context of the model run. If not specified, it is set to the latest block of the chain.
+  -c CHAIN_ID, --chain_id CHAIN_ID
+                        [OPTIONAL] The chain ID. Defaults to 1.
+  -i INPUT, --input INPUT
+                        [OPTIONAL] Input JSON or if value is "-" it will read input JSON from stdin.
+  -v MODEL_VERSION, --model_version MODEL_VERSION
+                        [OPTIONAL] Version of the model to run. Defaults to latest.
+  --provider_url_map PROVIDER_URL_MAP
+                        [OPTIONAL] JSON object of chain id to Web3 provider HTTP URL. Overrides settings in env vars.
+  --api_url API_URL     [OPTIONAL] Credmark API url. Defaults to the standard API gateway. You do not normally need to set this.
+  --api_url API_URL     [OPTIONAL] Credmark API url
+```
+
+To call any model we need to provide below parameters:
+
+- `-b` or `–block_number` : to define against which block number the model should run. If not provided it is set to latest block number available for the chain.
+- `-i` or `–input` : to provide input for a model in a predefined structure.(you can run command `credmark-dev list --manifests` to see the input format required for each model. See example below). If not provided it will default to “{}”.
+Model-slug: Name of the model (slug) to call the model.
+
+Note:  if chain ID is not mentioned explicitly in the parameter,  it defaults to 1. If the model is using web 3 instance then chain id (and blockchain) will be picked from the .env file we defined during setup (refer to “configure environment variable” section). If the model is using Credmark database then, by default, it will refer to the Ethereum blockchain.
+
+See the example below. Here, we are running the model “cmk-circulating-supply” at block_number 14000000.
+```
+>Credmark-dev run -b 14000000 cmk-circulating-supply -i "{}"
+
+2022-03-04 11:21:00,362 - credmark.model.engine.model_loader - INFO - Loading manifest from model_paths: ['models']
+{"slug": "cmk-circulating-supply", "version": "1.0", "output": {"result": 28314402605762084044696668}, "dependencies": {"cmk-total-supply": {"1.0": 1}, "cmk-circulating-supply": {"1.0": 1}}}
+```
+
+## `list`command
+Below `-h` command shows the details of options available for list commands.
+```
+>credmark-dev list -h
+
+usage: credmark-dev list [-h] [--manifests] [--json]
+
+optional arguments:
+  -h, --help   show this help message and exit
+  --manifests
+  --json
+```
+Note: You can also run `list-models` command alternatively.
+
+Example below shows simple output (list of all models and their version) of list command:
+```
+>credmark-dev list -h
+
+2022-03-04 11:42:46,691 - credmark.model.engine.model_loader - INFO - Loading manifest from model_paths: ['models']
+
+Loaded models:
+
+ - var: ['1.0']
+ - cmk-total-supply: ['1.0']
```

Comment:
I don't think we should list a lot of models here or people will think it's the full list. I think just show a few and a "..." gap.